pkg/operator/starter.go

```diff
@@ -18,6 +18,7 @@ import (
 	operatorv1informers "github.com/openshift/client-go/operator/informers/externalversions"
 	"github.com/openshift/library-go/pkg/controller/controllercmd"
 	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/unsupportedconfigoverridescontroller"
```

Comment:
HOLY MOLY! That's one long name :( I probably would've aliased this to ucoc or something shorter but that's more style than actual problem.